### ISSUE TITLE
🐛 BUG: Get tokens

### ISSUE THREAD

**Issue by namasiK** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#267]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 Build 1909]
 - Version [e.g. 1.0.0-rc.1]

**Additional context**

**Comment by namasiK** [comment]
STRUCK at GET TOKENS,

I Had tried every steps on making my own google Assistant but when the steps says to relaunch the Assistance after locating the path, For the "GET TOKENS" The browser window is supposed to be opening automatically but it is not opening in my case.

Please find the solution for this..! I love your work between

![Untitled](https://user-images.githubusercontent.com/81893030/113498377-029de780-952a-11eb-8685-c4c562f045a4.jpg)

**Comment by Melvin-Abraham** [comment]
@namasiK Can you make sure if your browser is minimized? If not, make sure you have a default browser set on your system.

**Comment by namasiK** [comment]
Yes the browser is minimized and default browser is set to Google Chrome
![image](https://user-images.githubusercontent.com/81893030/113660137-16c81d00-96c1-11eb-93da-2c217eaf8f97.png)

**Comment by Melvin-Abraham** [comment]
I am not sure about the cause of this issue but I think a good way to deal with this would be to allow the users to copy the Authorization link and let them paste on the browser manually.

**Comment by ninjaeon** [comment]
I had the same issue.  This was previously reported.  Reading through what others said, after restarting Windows the browser window will open.  That's what happened for me, the window automatically opened immediately after Windows loaded after restarting.  Some mentioned that logging off and logging back in also resolves this, but I didn't test that.
